metalware-new/src/commands/build.rb

```diff
@@ -1,6 +1,7 @@
 
 require 'active_support/core_ext/string/strip'
 
+require 'commands/basecommand'
```

Comment:
The usual convention in Ruby is to name the class/module using `CamelCase` (which you've done) and then the file using `underscore_case` - very minor but can we name the file `base_command` :slightly_smiling_face:?